plugins/GSdx/Renderers/DX11/GSRendererDX11.cpp

```diff
@@ -229,11 +229,13 @@ void GSRendererDX11::EmulateTextureShuffleAndFbmask()
 		case ACC_BLEND_MEDIUM_D3D11:
 			// Enable Fbmask emulation excluding triangle class because it is quite slow.
 			// Exclude 0x80000000 because Genji needs sw blending, otherwise it breaks some effects.
+			// (FIXME: It should (hopefully) be safe to remove the check for 0x80000000 now)
```

Comment:
I never had much love for it when I first added the check :p, you can purge it, Genji seems fine now.